modules/nodes-pods-configuring-pod-critical.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+
+[id='nodes-pods-configuring-critical_{context}']
+= Preventing pod removal using critical pods
+
+There are a number of core components, such as Heapster and DNS, that are critical to a fully functional cluster,
+but, run on a regular cluster node rather than the master. A cluster may stop working properly if a critical add-on is evicted.
+
+The descheduler does not remove critical pods.
```

Comment:
I would make this more generic.  I would say "Pods marked as critical are not able to be evicted".